pkg/interfaces/contracts/solidity-utils/misc/ICowConditionalOrder.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.8.0 <0.9.0;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+struct GPv2Order {
+    IERC20 sellToken;
+    IERC20 buyToken;
+    address receiver;
+    uint256 sellAmount;
+    uint256 buyAmount;
+    uint32 validTo;
+    bytes32 appData;
+    uint256 feeAmount;
+    bytes32 kind;
+    bool partiallyFillable;
+    bytes32 sellTokenBalance;
+    bytes32 buyTokenBalance;
+}
+
+/**
+ * @title Conditional Order Interface
+ * @author CoW Protocol Developers + mfw78 <mfw78@rndlabs.xyz>
```

Comment:
We haven't been using title, but author is fine here. Would also add a permalink to the reference code.